temoa/model_checking/commodity_graph.py

```
@@ -228,6 +235,114 @@ def generate_commodity_graph(
+
+    dot_content = results_dot_fmt.format(**args)
+
+    output_name = output_dir / f'results_{region}_{period}'
+    dot_file = output_name.with_suffix('.dot')
+    svg_file = output_name.with_suffix('.svg')
+
+    with open(dot_file, 'w') as f:
+        f.write(dot_content)
+
+    try:
+        subprocess.call(['dot', '-Tsvg', f'-o{svg_file}', str(dot_file)])
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)